### PR TITLE
Set phpstan to level 1

### PR DESCRIPTION
**Pull request by jjanvier**
This fixes possibly undefined variables.

### CODE REVIEW

File: src/Akeneo/Pim/Enrichment/Component/Product/Factory/Value/DateValueFactory.php [hunk 1]
```diff
@@ -34,29 +34,22 @@ protected function prepareData(AttributeInterface $attribute, $data, bool $ignor
...
         try {
             $date = new \DateTime($data);
 
             if (!preg_match('/^\d{4}-\d{2}-\d{2}/', $data)) {
-                $this->throwsInvalidDateException($attribute, $data);
+                throw $this->buildInvalidDateException($attribute, $data);
             }
         } catch (\Exception $e) {
-            $this->throwsInvalidDateException($attribute, $data);
+            throw $this->buildInvalidDateException($attribute, $data);
```
Comment (phaseinducer) [inline src/Akeneo/Pim/Enrichment/Component/Product/Factory/Value/DateValueFactory.php]: You should pass the previous exception here.
I know it implies to improve the named constructors of `InvalidPropertyException` so maybe not the right time to do it, as you want.

File: src/Akeneo/Pim/Enrichment/Component/Product/Connector/Processor/Denormalizer/ProductProcessor.php [hunk 2]
```diff
@@ -267,4 +269,14 @@ protected function detachProduct(ProductInterface $product)
...
     }
+
+    private function skipItemAndReturnException(array $item, $message, \Exception $previousException = null): InvalidItemException
+    {
+        if ($this->stepExecution) {
+            $this->stepExecution->incrementSummaryInfo('skip');
+        }
+        $itemPosition = null !== $this->stepExecution ? $this->stepExecution->getSummaryInfo('item_position') : 0;
+        $invalidItem = new FileInvalidItem($item, $itemPosition);
+        return new InvalidItemException($message, $invalidItem, [], 0, $previousException);
```
Comment (mmetayer) [inline src/Akeneo/Pim/Enrichment/Component/Product/Connector/Processor/Denormalizer/ProductProcessor.php]: missing empty line :)

Comment (jjanvier) [reply src/Akeneo/Pim/Enrichment/Component/Product/Connector/Processor/Denormalizer/ProductProcessor.php]: fixed

**Comment by jjanvier** [comment]
just added a missing empty line before return in the last commit
merging

File: src/Akeneo/Pim/Enrichment/Component/Product/Factory/Value/DateValueFactory.php (hunk 1 above)
Comment (jjanvier) [reply src/Akeneo/Pim/Enrichment/Component/Product/Factory/Value/DateValueFactory.php]: yep, I won't do that here
it's not the purpose of this PR

**Pull request merged**